documentation/configuration.markdown

```diff
@@ -181,6 +195,33 @@ The default Logback configuration file is at `/etc/puppetserver/logback.xml` or
 
 Puppet Server relies on `logrotate` to manage the log file, and installs a configuration file at `/etc/logrotate.d/puppetserver` or `/etc/logrotate.d/pe-puppetserver`.
 
+### HTTP Traffic
+
+Puppet Server logs HTTP traffic in a format similar to Apache, and to a separate
+file than the main log file. By default, this is located at
+`/var/log/puppetserver/puppetserver-access.log` (open source releases) and
```

Comment:
This one should be `/var/log/puppetlabs/puppetserver/puppetserver-access.log` now, right?